specification/cognitiveservices/ContentSafety/models.tsp

```diff
@@ -146,50 +141,142 @@ model TextBlocklist {
+@resource("blocklistItems")
 @parentResource(TextBlocklist)
-model TextBlockItem {
-  @doc("Block Item Id. It will be uuid.")
-  @key("blockItemId")
-  @visibility("read", "create", "query")
+model TextBlocklistItem {
+  @doc("The service will generate a BlocklistItemId, which will be a UUID.")
+  @key("blocklistItemId")
+  @visibility("read")
   @maxLength(64)
-  blockItemId: string;
+  blocklistItemId: string;
 
-  @doc("Block item description.")
+  @doc("BlocklistItem description.")
   @maxLength(1024)
   description?: string;
 
-  @doc("Block item content.")
+  @doc("BlocklistItem content.")
   @maxLength(128)
   text: string;
 }
 
-@doc("Block item info in text blocklist.")
-model TextBlockItemInfo {
-  @doc("Block item description.")
+@doc("The request to add blocklistItems to a text blocklist.")
+model AddOrUpdateTextBlocklistItemsOptions {
+  @doc("Array of blocklistItems to add.")
+  blocklistItems: TextBlocklistItem[];
+}
+
+@doc("The response of adding blocklistItems to the text blocklist.")
+model AddOrUpdateTextBlocklistItemsResult {
+  @doc("Array of blocklistItems have been added.")
+  blocklistItems?: TextBlocklistItem[];
+}
+
+@doc("The request to remove blocklistItems from a text blocklist.")
+model RemoveTextBlocklistItemsOptions {
+  @doc("Array of blocklistItemIds to remove.")
+  blocklistItemIds: string[];
+}
+
+@added(ContentSafety.Versions.v2023_10_15_Preview)
+@doc("Project is an entity to manage deployment with specified policy.")
+@resource("projects")
+model Project {
+  @doc("Project name.")
+  @pattern("^[a-zA-Z][0-9A-Za-z-]*[a-zA-Z]$")
+  @key("projectName")
+  @visibility("read", "query")
+  @maxLength(255)
+  @minLength(2)
+  projectName: string;
+
+  @doc("Project description.")
   @maxLength(1024)
   description?: string;
 
-  @doc("Block item content.")
-  @maxLength(128)
-  text: string;
+  @doc("Project modality.")
+  @visibility("read", "query", "create")
+  modality: ContentModality;
+
+  @doc("Project display name.")
+  @maxLength(255)
+  @minLength(2)
+  displayName?: string;
+
+  @doc("Project created time.")
+  @visibility("read", "query")
+  createdAt?: utcDateTime;
+
+  @doc("Project last modified time.")
+  @visibility("read", "query")
+  lastModifiedAt?: utcDateTime;
+
+  @doc("Deployed text policy.")
+  textPolicy?: TextPolicy;
+
+  @doc("Deployed image policy.")
+  imagePolicy?: ImagePolicy;
+}
+
+@added(ContentSafety.Versions.v2023_10_15_Preview)
+@doc("Modality type.")
+enum ContentModality {
+  @doc("Text modality.")
+  Text,
+  @doc("Image modality.")
+  Image,
+}
+
+@added(ContentSafety.Versions.v2023_10_15_Preview)
+@doc("TextCategoryPolicy. Content which severity less than the acceptLessThan will be accepted.")
+model TextCategoryPolicy {
+  @doc("Text category.")
+  category: TextCategory;
+
+  @doc("Severity less than or equal to the threshold will be treated as accepted for the category. The range is the severity value. Current the value could be: 0,2,4,6.")
```

Comment:
```suggestion
  @doc("Severity less than or equal to the threshold will be treated as accepted for the category. The range is the severity value. Current the value could be: 0, 2, 4, or 6.")
```

Please these same changes to all similar occurances.

Reply:
Will do.

Reply:
updated